api/views/assets.py

```diff
@@ -434,10 +435,74 @@ def get(self, request):
 
         file = open(filename, "rb")
         response = FileResponse(
-            file, content_type="text/csv", filename='sample_import_file.csv'
+            file, content_type="text/csv", filename="sample_import_file.csv"
         )
         response[
             "Content-Disposition"
         ] = 'attachment; filename="sample_import_file.csv"'
 
         return response
+
+
+class ExportAssetsDetails(APIView):
+    permission_classes = [IsAuthenticated, IsAdminUser]
+    authentication_classes = (FirebaseTokenAuthentication,)
+
+    def get(self, request):
+        assets = models.Asset.objects.filter(
+            asset_location__name=request.user.location.name
+        )
+        serializer = AssetSerializer(assets, many=True)
+        if len(serializer.data) == 0:
+            return Response({"error": "You have no assets"}, status=400)
+        self.create_sheet(serializer.data)
+        return Response(
+            {
+                "Success": "Assets details have been exported successfully, Find file in root folder"
+            },
+            status=200,
+        )
```

Comment:
we should return the url since this message is to the FE

Reply:
@egichuri can you please explain what you mean? I dont understand what you mean returning url.

Reply:
A working url for the location of the export file since the user might want to download it. The same way we're doing it for the [import](https://github.com/AndelaOSP/art-backend/blob/20087a548d89ac318fd6dc8b60738531aaf6c5df/api/views/assets.py#L397)